Python/Tests/Utilities.Python/PythonVisualStudioApp.cs

```diff
@@ -228,7 +228,13 @@ public PythonVisualStudioApp(IServiceProvider site)
                             System.Threading.Thread.Sleep(500);
                         }
                     }
-                    _testExplorer = new PythonTestExplorer(this, element);
+                    Assert.IsTrue(element != null, "Missing Text Explorer window");
```

Comment:
BTW there's `Assert.IsNotNull` for cases like this

Reply:
thanks
i'll update it